app/structure/snake-loader.js

```diff
@@ -0,0 +1,28 @@
+import fs from "fs";
+
+export default class SnakeLoader {
+    static importAll() {
```

Comment:
why?

Reply:
Why?

Reply:
Making things global never give us good consequences except situation when function it is really simple and contains no application business logic. This class contains it so it will be better to not use static keyword, because in future it probably could contain state and our code will need refactor. I suppose that static keyword is bad for code in general. I don't like it :)

Reply:
> I don't like it :)

FTFY